billboardd/tklockinterface.cc

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * Billboard - Low Power Mode Standby Screen for the N9
+ * Webpage: http://thp.io/2012/billboard/
+ * Copyright (C) 2012, 2013, 2014 Thomas Perl <thp.io/about>
+ * Copyright 2015 Elliot Wolk <elliot.wolk@gmail.com>
```

Comment:
Feel free to just have your line, as I didn't do anything in this file (yet). Also use "Copyright (C) 2015" (note the " (C)") in order to be consistent with the rest of the copyright notices.